fuzzing build faliure

https://oss-fuzz-build-logs.storage.googleapis.com/log-2b2be386-7e9d-4a34-a839-7ec37794b7f9.txt

```
Step #3 - "compile-libfuzzer-address-x86_64": Building fuzz_g1_add
Step #3 - "compile-libfuzzer-address-x86_64": + [[ address = *coverage* ]]
Step #3 - "compile-libfuzzer-address-x86_64": + cd /root/go/src/github.com/ethereum/go-ethereum/tests/fuzzers/bls12381
Step #3 - "compile-libfuzzer-address-x86_64": + go-fuzz -func FuzzG1Add -o /work/fuzz_g1_add.a .
Step #3 - "compile-libfuzzer-address-x86_64": # github.com/ethereum/go-ethereum/tests/fuzzers/bls12381
Step #3 - "compile-libfuzzer-address-x86_64": ./bls12381_fuzz.go:201:42: gnarkScalar.SetBigInt(s).FromMont undefined (type *fr.Element has no field or method FromMont)
Step #3 - "compile-libfuzzer-address-x86_64": ./bls12381_fuzz.go:220:27: not enough arguments in call to cp.MultiExp
Step #3 - "compile-libfuzzer-address-x86_64": 	have ([]"github.com/consensys/gnark-crypto/ecc/bls12-381".G1Affine, []fr.Element)
Step #3 - "compile-libfuzzer-address-x86_64": 	want ([]"github.com/consensys/gnark-crypto/ecc/bls12-381".G1Affine, []fr.Element, ecc.MultiExpConfig)
Step #3 - "compile-libfuzzer-address-x86_64": 2023/02/11 06:11:04 failed to build packages:exit status 2
Step #3 - "compile-libfuzzer-address-x86_64": ********************************************************************************
Step #
```
Most likely fallout from https://github.com/ethereum/go-ethereum/pull/26545

--- a/tests/fuzzers/bls12381/bls12381_fuzz.go
+++ b/tests/fuzzers/bls12381/bls12381_fuzz.go
@@ -218,7 +218,7 @@ func FuzzCrossG1MultiExp(data []byte) int {
 
 	// gnark multi exp
 	cp := new(gnark.G1Affine)
-	cp.MultiExp(gnarkPoints, gnarkScalars, ecc.MultiExpConfig{NbTasks: 1})
+	cp.MultiExp(gnarkPoints, gnarkScalars, ecc.MultiExpConfig{})
 
 	// compare result
 	if !(bytes.Equal(cp.Marshal(), g1.ToBytes(&kp))) {

--- a/tests/fuzzers/bls12381/bls12381_fuzz.go
+++ b/tests/fuzzers/bls12381/bls12381_fuzz.go
@@ -26,6 +26,7 @@ import (
 	"io"
 	"math/big"
 
+	"github.com/consensys/gnark-crypto/ecc"
 	gnark "github.com/consensys/gnark-crypto/ecc/bls12-381"
 	"github.com/consensys/gnark-crypto/ecc/bls12-381/fp"
 	"github.com/consensys/gnark-crypto/ecc/bls12-381/fr"
@@ -198,7 +199,7 @@ func FuzzCrossG1MultiExp(data []byte) int {
 		}
 		gethScalars = append(gethScalars, s)
 		var gnarkScalar = &fr.Element{}
-		gnarkScalar = gnarkScalar.SetBigInt(s).FromMont()
+		gnarkScalar = gnarkScalar.SetBigInt(s)
 		gnarkScalars = append(gnarkScalars, *gnarkScalar)
 
 		gethPoints = append(gethPoints, new(bls12381.PointG1).Set(kp1))
@@ -217,7 +218,7 @@ func FuzzCrossG1MultiExp(data []byte) int {
 
 	// gnark multi exp
 	cp := new(gnark.G1Affine)
-	cp.MultiExp(gnarkPoints, gnarkScalars)
+	cp.MultiExp(gnarkPoints, gnarkScalars, ecc.MultiExpConfig{NbTasks: 1})
 
 	// compare result
 	if !(bytes.Equal(cp.Marshal(), g1.ToBytes(&kp))) {

